<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>WkfCommentPrefWidget</class>
 <widget class="QWidget" name="WkfCommentPrefWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>365</width>
    <height>299</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Comment Bubbles</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_2">
   <item>
    <widget class="QGroupBox" name="enableGroupBox">
     <property name="title">
      <string>Enable Comment Bubbles</string>
     </property>
     <property name="checkable">
      <bool>true</bool>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <layout class="QFormLayout" name="formLayout">
        <item row="0" column="0">
         <widget class="QLabel" name="timestampLabel">
          <property name="text">
           <string>Include Timestamp</string>
          </property>
         </widget>
        </item>
        <item row="0" column="1">
         <widget class="QCheckBox" name="timestampCheckBox">
          <property name="text">
           <string/>
          </property>
         </widget>
        </item>
        <item row="1" column="0">
         <widget class="QLabel" name="timeoutLabel">
          <property name="text">
           <string>Timeout (sec)</string>
          </property>
         </widget>
        </item>
        <item row="1" column="1">
         <widget class="QSpinBox" name="timeoutSpinBox">
          <property name="minimum">
           <number>1</number>
          </property>
          <property name="maximum">
           <number>10000</number>
          </property>
          <property name="value">
           <number>1</number>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>enableGroupBox</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
